src/plugins/data_table/data_table.js

```diff
@@ -2,30 +2,167 @@ var LocalDataTable = (function() {
 
   var Base = Backdraft.plugin("Base");
 
+  var DEFAULT_JST_DELIMS = {
+    evaluate    : /<%([\s\S]+?)%>/g,
+    interpolate : /<%=([\s\S]+?)%>/g,
+    escape      : /<%-([\s\S]+?)%>/g
+  };
+
+  var DataTableFilter = Base.View.extend({
+    filterTemplate : _.template('\
+      <div class="toggle-filter-button" data-toggle="dropdown">\
+        <span class="<%= filterButtonClass %>"></span>\
+      </div>\
+    ', null, DEFAULT_JST_DELIMS),
+
+    filterMenuTemplate : _.template('\
+      <div class="filterMenu dropdown-menu <%= listClass %>">\
+        <% if (filter.type === "string") { %>\
+          <input class="filter-string" id ="value" type="text" placeholder="Search <%= title %>" />\
+        <% } else if (filter.type === "numeric") { %>\
+          <ul>\
+            <li> &gt; <input id="gt" class="filter-numeric" type="text" /></li> \
+            <li> &lt; <input id="lt" class="filter-numeric" type="text"/></li> \
+            <li> = <input id="eq" class="filter-numeric" type="text" /></li> \
+          </ul>\
+        <% } else if (filter.type === "list") { %>\
+          <ul>\
+            <% _.each(filter.options, function(element, index) { %>\
+              <li>\
+                <label>\
```

Comment:
For better usability, and in case you need to programmatically check a checkbox later, it's generally best practice to wrap checkboxes and their individual labels (like you did here). But also to give it a markup like this:

<label for="someUniqueId"><input type="checkbox" id="someUniqueId" name="somename" />Text that is displayed to the user.</label>

Without the "label for", the browser is interpreting that "this label is for the input box between the label tags." Adding the "for" attribute explicitly tells the browser, "this label is for this checkbox."